src/index.js

```diff
@@ -1,4 +1,168 @@
 'use strict'
 
-exports.IPLDService = require('./ipld-service')
-exports.resolve = require('./resolve')
+const Block = require('ipfs-block')
+const pull = require('pull-stream')
+const traverse = require('pull-traverse')
+const utils = require('./utils')
+const CID = require('cids')
+const until = require('async/until')
+const IPFSRepo = require('ipfs-repo')
+const MemoryStore = require('../node_modules/interface-pull-blob-store/lib/reference.js')
```

Comment:
you can do `require('interface-pull-blob-store/lib/reference.js')` instead, no need for the full path